docs/csharp/language-reference/keywords/ref.md

```diff
@@ -115,13 +122,30 @@ When the caller stores the value returned by the `GetBookByTitle` method as a re
 
 [!code-csharp[csrefKeywordsMethodParams#6](~/samples/snippets/csharp/language-reference/keywords/in-ref-out-modifier/RefParameterModifier.cs#5)]
 
+## Ref struct types
+
+Adding the `ref` modifier to a `struct` declaration declares that instances of that type must be stack allocated. In other words, these types can never be created on the
```

Comment:
nit: "these types can never be created" -> "instances of that type can never be created"

Reply:
declaration declares is a bit funny. Can you use another verb perhaps like defines?